yarn.lock

```diff
@@ -920,37 +920,51 @@
     "@babel/helper-validator-identifier" "^7.14.0"
     to-fast-properties "^2.0.0"
 
-"@binance-chain/javascript-sdk@^2.14.4":
-  version "2.14.4"
-  resolved "https://registry.yarnpkg.com/@binance-chain/javascript-sdk/-/javascript-sdk-2.14.4.tgz#a68e2ff7efc7a346f7945b0eeb8ddc03aac0ec2c"
-  integrity sha512-v8eXyTXaD8/wPrWVqoJkd1y78erHuPvajEDrtnElU5Jf84iq2lbICOYsAU03IM6UfqT9dlo2UCBLt8usSQxmKQ==
-  dependencies:
-    "@babel/runtime" "^7.4.4"
-    "@ledgerhq/hw-transport-u2f" "^4.63.2"
-    "@ledgerhq/hw-transport-web-ble" "^4.63.2"
+"@binance-chain/javascript-sdk@^4.2.0":
+  version "4.2.0"
+  resolved "https://registry.yarnpkg.com/@binance-chain/javascript-sdk/-/javascript-sdk-4.2.0.tgz#9f05e39209d45344c998ad0fe9497f2533413545"
+  integrity sha512-TGBZ2M1e7XnDCsNfHvNf0Zj8diH81scUzhkiPJ+4cJRZmOZL4yBy6g9dhM4SLUAB3R8t87arexd2BD3C/sfjjQ==
+  dependencies:
+    "@babel/runtime" "^7.10.4"
+    "@ledgerhq/hw-transport-u2f" "^5.9.0"
+    "@ledgerhq/hw-transport-web-ble" "^5.9.0"
+    "@types/bech32" "^1.1.2"
+    "@types/big.js" "^4.0.5"
+    "@types/bn.js" "^4.11.6"
+    "@types/crypto-js" "^3.1.43"
+    "@types/elliptic" "^6.4.12"
+    "@types/jest" "^25.1.4"
+    "@types/ledgerhq__hw-transport" "^4.21.2"
+    "@types/lodash" "^4.14.149"
+    "@types/node" "^13.9.0"
+    "@types/pumpify" "^1.4.1"
+    "@types/tiny-secp256k1" "^1.0.0"
+    "@types/uuid" "^7.0.0"
     axios "^0.19.0"
     bech32 "^1.1.3"
     big.js "^5.2.2"
-    bip32 "^1"
+    bip32 "^2.0.5"
     bip39 "^3.0.2"
     bn.js "^4.11.8"
     camelcase "^5.3.1"
     crypto-browserify "^3.12.0"
     crypto-js "^3.1.9-1"
+    elliptic "^6.0.0"
+    eslint-utils "^1.4.2"
     events "^3.0.0"
     is_js "^0.9.0"
+    lodash "^4.17.19"
+    minimist "^1.2.5"
     ndjson "^1.5.0"
     protocol-buffers-encodings "^1.1.0"
-    pumpify "^1.5.1"
-    safe-buffer "^5.1.2"
+    pumpify "^2.0.1"
     secure-random "^1.1.2"
     tiny-secp256k1 "^1.1.3"
     url "^0.11.0"
     uuid "^3.3.2"
-    varstruct "^6.1.2"
     websocket-stream "^5.5.0"
   optionalDependencies:
-    "@ledgerhq/hw-transport-node-hid" "^4.63.2"
+    "@ledgerhq/hw-transport-node-hid" "^5.10.0"
```

Comment:
'--no-optional' is npm's flag, yarn uses '--ignore-optional', that's why this is still here ;)